commands/utils.py

```diff
@@ -105,7 +106,16 @@ def setup_templates():
             "aurora-postgres": templateEnv.get_template("addons/env/aurora-postgres.yml"),
             "redis": templateEnv.get_template("addons/env/redis-cluster.yml"),
             "s3": templateEnv.get_template("addons/env/s3.yml"),
+            "waf": templateEnv.get_template("env/addons/waf.yml"),
+            "parameters": templateEnv.get_template("env/addons/addons.parameters.yml"),
```

Comment:
I think these paths are wrong?

```suggestion
            "waf": templateEnv.get_template("addons/env/waf.yml"),
            "parameters": templateEnv.get_template("addons/env/addons.parameters.yml"),
```

Reply:
Nah they are correct. WAF applies to the ALB which is environment level.  I moved the templates back to this directory.

Reply:
I think you got `env` and `addons` the wrong way round though?